src/main/kotlin/com/openlattice/datastore/services/EntitySetService.kt

```diff
@@ -134,14 +137,14 @@ open class EntitySetService(
             )
 
             entityType.properties
+                    .parallelStream()
                     .map { propertyTypeId -> AclKey(entitySetId, propertyTypeId) }
-                    .onEach { aclKey ->
-                        authorizations.setSecurableObjectType(aclKey, SecurableObjectType.PropertyTypeInEntitySet)
-                    }
                     .forEach { aclKey ->
+                        authorizations.setSecurableObjectType(aclKey, PropertyTypeInEntitySet)
```

Comment:
these should be using the new bulk setSecurableObjectTypes + addPermissions calls, yeah?